specification/security/resource-manager/Microsoft.Security/stable/2021-06-01/settings.json

```diff
@@ -0,0 +1,300 @@
+          "Update a setting for subscription": {
+            "$ref": "./examples/Settings/UpdateSetting_example.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "updating settings about different configurations in security center",
+        "operationId": "Settings_Update",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingName"
+          },
+          {
+            "$ref": "#/parameters/Setting"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SettingsList": {
+      "type": "object",
+      "description": "Subscription settings list.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The settings list.",
+          "items": {
+            "$ref": "#/definitions/Setting"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "DataExportSettings": {
+      "type": "object",
+      "description": "Represents a data export setting",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Data export setting data",
+          "$ref": "#/definitions/DataExportSettingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Setting"
+        }
+      ],
+      "x-ms-discriminator-value": "DataExportSettings"
+    },
+    "AlertSyncSettings": {
+      "type": "object",
+      "description": "Represents an alert sync setting",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Alert sync setting data",
+          "$ref": "#/definitions/AlertSyncSettingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Setting"
+        }
+      ],
+      "x-ms-discriminator-value": "AlertSyncSettings"
+    },
+    "Setting": {
+      "type": "object",
+      "description": "The kind of the security setting",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "the kind of the settings string",
+          "enum": [
+            "DataExportSettings",
+            "AlertSuppressionSetting",
```

Comment:
Why is AlertSuppressionSetting not used in plural way? Other setting names DataExportSettings, AlertSyncSettings are plural.